build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+UNAME=$(uname -s)
+if [ "$UNAME" = "Linux" ]
+then
+    if dpkg -l python3-venv; then
+    echo "python3-venv is installed, skipping setup"
+    else
+    echo "Installing venv on Linux"
+    sudo apt-get install -y python3-venv
+    fi
+fi
+if [ "$UNAME" = "Darwin" ]
+then
+echo "Installing venv on Darwin"
+brew install python3-venv
+fi
```

Comment:
I don't think this a brew package that is available - should probably remove this